pkgs/development/tools/misc/cli11/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, cmake, gtest, python, boost }:
+
+stdenv.mkDerivation rec {
+  name = "cli11-${version}";
+  version = "1.7.1";
+
+  src = fetchFromGitHub {
+    owner = "CLIUtils";
+    repo = "CLI11";
+    rev = "v${version}";
+    sha256 = "0wddck970pczk7c201i2g6s85mkv4f2f4zxy6mndh3pfz41wcs2d";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  checkInputs = [ boost python ];
+
+  doCheck = true;
+
+  preConfigure = stdenv.lib.optionalString doCheck ''
```

Comment:
This `stdenv.lib.optionalString doCheck` only changes behavior if somebody changes the very line just above this one by setting `doCheck = false`, it won't have any effect when somebody sets `doCheck` from outside. So I think this condition can be removed.